src/api/app/models/workflow_run.rb

```diff
@@ -22,7 +22,7 @@ class WorkflowRun < ApplicationRecord
     :state, :status_options
   ].freeze
 
-  ALL_POSSIBLE_REQUEST_ACTIONS = ['all'] + ALLOWED_PULL_REQUEST_ACTIONS + ALLOWED_MERGE_REQUEST_ACTIONS
+  ALL_POSSIBLE_REQUEST_ACTIONS = ['all'] + ALLOWED_GITHUB_PULL_REQUEST_ACTIONS + ALLOWED_MERGE_REQUEST_ACTIONS
```

Comment:
What about gitea?

Reply:
We can add it for clarity and in case they change in the future. Even if we know now they are duplicated from Github ones:

``` 
ALL_POSSIBLE_REQUEST_ACTIONS = (['all'] + ALLOWED_GITHUB_PULL_REQUEST_ACTIONS + ALLOWED_MERGE_REQUEST_ACTIONS + ALLOWED_GITEA_PULL_REQUEST_ACTIONS).uniq
```